src/coreclr/clrdefinitions.cmake

```diff
@@ -120,9 +120,9 @@ endif(FEATURE_GDBJIT_LANGID_CS)
 if(FEATURE_GDBJIT_SYMTAB)
     add_definitions(-DFEATURE_GDBJIT_SYMTAB)
 endif(FEATURE_GDBJIT_SYMTAB)
-if(CLR_CMAKE_TARGET_UNIX)
+if(CLR_CMAKE_TARGET_LINUX)
```

Comment:
I wonder if this should actually be controlled by the cmake `FeatureXplatEventSource` property set  above.  That would keep them consistent.